roles/tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -50,6 +50,15 @@
   notify:
     - restart grafana-server
 
+- name: Configure admin password for grafana
```

Comment:
@mbukatov grafana.ini will be available at this location (/etc/tendrl/monitoring-integration/grafana/grafana.ini) only after monitorint-integration is installed, so please make sure that this script runs only after monitoring-integration is installed

Reply:
@rishubhjain installation of `tendrl-monitoring-integratino` package is the very first task in this particular file, so it should be ok